README.md

```diff
@@ -0,0 +1,7 @@
+# Doppler Contact Policies API
+
+Mini-API to allow users to configure Doppler's contact policies.
+
+See [implementation details](./docs/implementation-details.md).
+
+You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).
```

Comment:
What do you think about including the following information?

```suggestion
You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).

# CD
- Production environment
URL: https://apis.fromdoppler.com/contact-policies 
Updated when generating a new tag.

- Integration environment
URL: https://apisint.fromdoppler.net/contact-policies 
Updated when add changes into the INT branch

- QA environment
URL: https://apisqa.fromdoppler.net/contact-policies 
Updated when add changes to the main branch
```

Reply:
Mmm, ok.

```suggestion
You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).

## Environments 
          
### Production

URL: <https://apis.fromdoppler.com/contact-policies/>

The production environment is updated when a new `v1.*` tag is pushed.

### QA

URL: <https://apisqa.fromdoppler.net/contact-policies/>

The QA environment is updated when `main` branch is updated.

### INT

URL: <https://apisint.fromdoppler.net/contact-policies/>

The INT environment is updated when `INT` branch is updated.          
```

Reply:
Thanks @ms-darianbenito, I updated it based on your comment, with some format and style changes. Could you review it again?

Reply:
Great!